### ISSUE TITLE
Bug?

### ISSUE THREAD

**Issue by hoaug-tran** [NozakiYuu/FacebookWidget#1]
**Thank you for creating this amazing software!**

During use, I encountered the following problems, I don't know if it is a bug or not?

1. When I click the Start window button, the widget automatically hides and doesn't re-enable until I click on the FacebookWidget icon or End task and run it again. The same thing happens when pressing the Search button.

2. When I click the Show desktop button in the right corner of the Taskbar, I have the same problem as above. (I see that this bug has been fixed but I see that it is still not fixed: _The widget disappears when focusing on taskbar or clicking the Show Desktop button. This is Windows' restriction and I will find a way to fix it later. If you encounter this error, just press the Alt + Tab combination and click on the widget window. Fixed in version 1.1!)_ 

3. Information about messages (unread messages) not working even though I have correctly filled in Cookie and Facebook ID.

Hope you can help, thank you.

**Comment by Serena1432** [comment]
Thank you for reporting problems that you're encountering.

1. This is Windows' restrictions, and I'm currently unable to fix it. This application was used to disappear even when clicking or focusing on the taskbar because of these restrictions, but I fixed the taskbar-clicking thing. Sadly, the Start and Search menu aren't.
2. What Windows version are you using? The widget still shows up when pressing the Show Desktop button on Windows 11.
3. This may be because of the update time. To prevent your account from being disabled by Facebook, the update time is quite slow (90 seconds for each update), so it can take up to 180 seconds (3 minutes) to update the unread messages count. I used to reduce the update time before, but later Facebook disabled my account temporarily, so no.

But I agree with you that this application still has at least one bug. I will try to fix it later.

Anyways, **thank you for using my application!** Have a nice day!
